Cms_frontend/src/components/Editor/EditorSidebar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import styled from 'styled-components';
+
+
+const Container = styled.div`
+  background: #eeeeee;
+  width: 15%;
+  overflow-y: scroll;
```

Comment:
i think viewport width is better because % takes % of parent's width and sometimes the parent's width isnt 100% of screen size, overflow-y: i guess we can add a scroll but, imo we wont have enough items and even if we do have that many items im not sure if a scrollbar is the best UI design for it

Reply:
Just included the scrollbar because that's what we had on our figma